doc/source/security.md

```diff
@@ -4,7 +4,64 @@ The information in this document focuses primarily on cloud based deployments. F
 
 Brad Geesamen gave a wonderful talk titled [Hacking and Hardening Kubernetes by Example](https://kccncna17.sched.com/event/CU6z/hacking-and-hardening-kubernetes-clusters-by-example-i-brad-geesaman-symantec) at Kubecon NA 2017. You can [watch the talk](https://www.youtube.com/watch?v=vTgQLzeBfRU) or [read the slides](https://schd.ws/hosted_files/kccncna17/47/Hacking%20and%20Hardening%20Kubernetes%20By%20Example%20v1.pdf). Highly recommended that you do so to understand the security issues you are up against when using Kubernetes to run JupyterHub.
 
-## Secure the access to Helm
+## HTTPS
+
+Enabling HTTPS for your hub is pretty easy, since we automatically integrate with [Let's Encrypt](https://letsencrypt.org/).
```

Comment:
can we avoid using words like "easy" in the documentation? This is a highly user-specific word :-)

Reply:
instead maybe just "This section describes how to enable HTTPS on your JupyterHub. The easiest way to do so is by using [Let's Encrypt](https://letsencrypt.org/), though it is also possible to set up your own HTTPS credentials.".